axelrod/tests/unit/test_filters.py

```diff
@@ -16,6 +16,14 @@ class TestStrategy(object):
             'makes_use_of': ['game', 'length']
         }
 
+    class TestStrategy2(object):
+        classifier = {
+            'stochastic': True,
+            'inspects_source': False,
+            'memory_depth': 'infinity',
```

Comment:
The scenario where a strategy has `memory_depth': 'infinity'` is precisely what this test is for. There's a line in the filter that catches it and converts it to a float and it's that line that comes up on the coverage report as not covered.

I think I had to add the line to cope with one or two strategies that have this entry and were causing the API to fall over.